projects/worker-basic/manifest.yml

```diff
@@ -0,0 +1,9 @@
+packages:
+  __APP_PACKAGE__:
+    actions:
+      worker:
+        function: 'worker.js'
```

Comment:
Maybe we should use descriptive names, so if you have multiple examples/ workers open in your IDE, they aren't all named `worker.js`.

E.g. `example-worker-basic.js`.

Reply:
+1